redis/src/main/scala/zio/redis/Output.scala

```diff
@@ -2,6 +2,7 @@ package zio.redis
 
 import zio.Chunk
 import zio.duration._
+import zio.redis.api.Type
```

Comment:
There should be no need for this as it is imported implicitly through package object inheritance.